ansible_collections/arista/avd/roles/eos_l3ls_evpn/templates/fabric/mlag/leaf-mlag-spanning-tree.j2

```diff
@@ -1,4 +1,8 @@
 {# Leaf mlag disable spanning tree #}
 {% if leaf.mlag == true %}
+{%     if type == 'l3leaf' %}
   no_spanning_tree_vlan: 4093-4094
-{% endif %}
\ No newline at end of file
+{%     else type == 'l2leaf' %}
```

Comment:
Sorry I was not clear. This should be`{%     else %}`.
Current code is not valid. Remember to test !

Reply:
Tested
no spanning-tree vlan-id 4093-4094 - L3leaf
no spanning-tree vlan-id 4094 - L2leaf